docker-compose.yml

```diff
@@ -17,6 +17,30 @@ services:
     env_file:
       - ./local.env
 
+  test:
+    build: .
+    depends_on:
+      - ssp-hub.local
+      - ssp-hub-sp.local
+      - test-browser
+    environment:
+      - COMPOSER_CACHE_DIR=/composer
```

Comment:
It doesn't seem to. The PHPUnit tests still run even if that's not specified.

Interestingly, it appears that the production code defaults HUB_MODE to false, but the tests default it to true (and only run if it's true). So... leaving it unset doesn't seem to affect the tests, but it may be worth bringing those defaults into alignment with each other.

@Baggerone, do you happen to know if the default for HUB_MODE (here in ssp-base) is intentionally different between the actual code and the tests?

Reply:
IIrc, most instances will not be a hub, so the HUB_MODE = false default covers most cases.  
But, the tests were all for testing a Hub setup. So, they needed HUB_MODE to be true.